### PR TITLE
Blog post about Digital Transformation

### PR DESCRIPTION
**Pull request by shouqs**
a blog post about Digital transformation tips.

### CODE REVIEW

**Review by Myrthi74 — DISMISSED** [review]
@shouqs nur das "Arbeiten Sie Agile" stört mich, hatte ich im doc auf "agil" geändert und würde ich jetzt wieder ändern. Ansonsten passt der Text für mich.

**Review by shouqs — COMMENTED** [review]
I added Lukas edits

**Comment by shouqs** [comment]
@lukaseppler can you approve the request? I added all your comments form the google doc

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Die digitale Transformation ist eine Herausforderung
+
+Der Wandel betrifft Menschen, Kultur und Prozesse. Bei der digitalen Transformation geht es nicht nur um Technologie. Es geht darum, wie Sie Ihr Unternehmen so verändern, dass es in Zukunft kundenorientierter und agiler wird. Um dieses Ziel zu erreichen, müssen Sie verstehen, welchen Nutzen Ihr Unternehmen aus der digitalen Transformation zieht und wo es derzeit steht.
+Dies kann durch die Betrachtung von vier Dimensionen erreicht werden: 
+Prozesse (Modernisieren Sie Legacy-Prozesse, beschleunigen Sie effiziente Workflows, stärken Sie die Sicherheit und steigern Sie Ihre Produktivität)
+Menschen (Übergang von einzelnen Mitarbeitern zu vernetzten Teams)
+Kultur (Übergang von einer Befehls- und Kontrollkultur zu einer gemeinsamen Identität)
+Technologie (Übergang von monolithischen IT-Systemen zu Microservices)
```
Comment (lukaseppler) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:17]:
```suggestion

* Menschen (Übergang von einzelnen Mitarbeitern zu vernetzten Teams)
* Kultur (Übergang von einer Befehls- und Kontrollkultur zu einer gemeinsamen Identität)
* Technologie (Übergang von monolithischen IT-Systemen zu Microservices)

```

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Lösen Sie das richtige Problem
+
+Der erste Schritt bei jedem Projekt zur digitalen Transformation besteht darin, das Problem, das Sie lösen wollen, zu definieren und nicht einfach mit einer Idee zu beginnen. Deshalb ist es wichtig, dass Sie Ihr Team zusammenbringen und ihm Zeit geben, ein Brainstorming über die Art der Probleme zu machen, mit denen es konfrontiert ist. Danach gilt es herauszufinden, wie Technologie zur Lösung dieser Probleme beitragen könnte. Der Schlüssel dazu ist die Einsicht, dass es bei der digitalen Transformation nicht darum geht, Menschen zu ersetzen, sondern darum, menschliche Fähigkeiten zu entwickeln und zu verbessern. Besonders am Anfang lassen sich zum Teil mit einfachen Schritten grosse Vereinfachungen erreichen.
```
Comment (lukaseppler) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:21]: I think all other articles use informal tenses (Du). I don't particularly mind, but I think the web site and blog should have the same approach to talking with people. What do you think?

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Die Verwaltung von Talenten und der Aufbau digitaler Fähigkeiten sind zwei Möglichkeiten, wie sich Unternehmen auf zukünftiges Wachstum vorbereiten können. Das ist unabhängig davon, ob sie die Implementierung von KI planen oder lediglich ihre IT-Infrastruktur aktualisieren. Unternehmen sollten auch in die Schulung ihrer Mitarbeitenden investieren, damit diese die Technologie effektiver nutzen können. So wird sichergestellt, dass alle Zugang zu den Tools haben, die für die Ausführung bestimmter Aufgaben (wie die Verarbeitung von Kundendaten) benötigt werden.
+
+## Finden Sie die richtigen Technologie- und Lösungspartner
+
+Sie werden Technologie- und Lösungspartner finden wollen, die Ihre Strategie der digitalen Transformation unterstützen können. Im Folgenden finden Sie einige Tipps, wie Sie dies tun können:
+Verstehen Sie die Vorteile der Zusammenarbeit mit einem Technologiepartner: Ein Partner kann Ihrem Unternehmen helfen, Zugang zu den neuesten Spitzentechnologien zu erhalten, aber er verfügt auch über umfassende Erfahrung mit der Implementierung neuer Lösungen, die sich in anderen Unternehmen bewährt haben.
```
Comment (lukaseppler) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:27]: If you want to make paragraphs, you need an empty line inbetween. There's a preview option top right

Comment (lukaseppler) [reply content/posts/die-digitale-transformation-ist-eine-reise.md:27]: Or even stars for a bullet list (as above)

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+Innovation (Erstellung eines Machbarkeitsnachweises und Validierung des Geschäftsszenarios).
+Inkubation (Einführung eines minimal realisierbaren Produkts und Testen und Lernen mit agilen Sprints).
+Industrialisierung (Ausführen der Technologie und des Geschäftsprozesses in grossem Massstab). 
+Nach einer ersten Welle von Anwendungsfällen wiederholt sich der Prozess mit der nächsten Welle – und so weiter. Was in den verschiedenen Phasen umgesetzt wird ist nicht nur eine Frage des gesunden Menschenverstandes, sondern auch entscheidend, um die Zustimmung Ihres Teams zu erhalten und die Kosten überschaubar zu halten. Wenn Sie neue Technologien ausprobieren ohne zunächst zu verstehen, warum sie für Ihr spezielles Geschäftsproblem oder Ihre Geschäftseinheit geeignet sind, könnten Sie viel mehr als nötig in Testversionen investieren, die nie von jemandem im Team verwendet werden (und daher nutzlos sind). Andererseits sollten Sie, wenn Sie bahnbrechende Innovationen wie Chatbots oder künstliche Intelligenz (KI) testen, nicht einfach das für bare Münze nehmen, was deren Anbieter Ihnen sagen. 
+
+## Arbeiten Sie Agile
```
Comment (lukaseppler) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:46]:
```suggestion
## Arbeiten Sie Agil
```

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+## Vergessen Sie nicht die Menschen und die Kultur
+
+Dies ist der wichtigste Aspekt der Innovation, insbesondere wenn es um die digitale Transformation geht. Menschen sind nicht nur eine Ressource, sondern auch ein Vermögenswert. Der Erfolg eines Unternehmens hängt von seinen Mitarbeitern ab: Sie sind die Hauptakteure bei der Gestaltung der Unternehmenskultur und tragen zum Erfolg oder Misserfolg des Unternehmens bei. Sie können noch so tolle Ideen haben, wenn Sie kein Team haben, mit dem sie umgesetzt werden können, werden sie nicht zum Tragen kommen.
+Die digitale Transformation ist eine Herausforderung, aber wenn man sie Schritt für Schritt angeht, kann man seine Ziele erreichen. Die digitale Transformation ist eine Reise, kein Ziel.
```
Comment (lukaseppler) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:54]:
```suggestion
Die digitale Transformation ist eine Herausforderung, aber wenn man sie Schritt für Schritt angeht, kann man seine Ziele erreichen. Die digitale Transformation selbst ist eine Reise, kein Ziel an sich.
```
I find it confusing to first say you can reach your goals, and then that transformation is not a goal but a path. I think my suggestion makes that clearer.

**Review by lukaseppler — DISMISSED** [review]
I like the text and think it's very close to publishable. In that sense, see as my comments as a "concern", not an "objection".

File: content/posts/die-digitale-transformation-ist-eine-reise.md (hunk 2 above)
Comment (shouqs) [reply content/posts/die-digitale-transformation-ist-eine-reise.md:23]: we agreed to keep it in Sie format

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+## Testen und pilotieren Sie neue Technologien, aber nur dort, wo es sinnvoll ist
+
+Die digitale Transformation ist kein Projekt, das für alle passt. Sie erfordert eine sorgfältige Berücksichtigung des Kontexts und der Kultur Ihres Unternehmens sowie der Bedürfnisse und Fähigkeiten Ihrer Mitarbeiter. Ihre Lösung kann nicht einfach lauten: "Wir setzen Technologie ein, um dieses Problem zu lösen." Sie müssen sich Gedanken darüber machen, welche Art von Technologie für Ihre spezielle Situation am besten geeignet ist, wer sie nutzen wird und wie diese Personen mit ihr interagieren werden.
+Wir arbeiten mit Unternehmen zusammen, um wirkungsvolle Anwendungsfälle zu identifizieren, zu priorisieren und umzusetzen. Auf diese Weise können Kunden schnell Werte schaffen und gleichzeitig demonstrieren, was digitale Technologien und Arbeitsweisen für das Unternehmen leisten können. Der Prozess umfasst drei Phasen: 
+Innovation (Erstellung eines Machbarkeitsnachweises und Validierung des Geschäftsszenarios).
```
Comment (shouqs) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:43]: *

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+Inkubation (Einführung eines minimal realisierbaren Produkts und Testen und Lernen mit agilen Sprints).
```
Comment (shouqs) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:44]: *

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+Industrialisierung (Ausführen der Technologie und des Geschäftsprozesses in grossem Massstab).
```
Comment (shouqs) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:45]: *

**Review by Myrthi74 — APPROVED** [review]
Passt für mich

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+Die Verwaltung von Talenten und der Aufbau digitaler Fähigkeiten sind zwei Möglichkeiten, wie sich Unternehmen auf zukünftiges Wachstum vorbereiten können. Das ist unabhängig davon, ob sie die Implementierung von KI planen oder lediglich ihre IT-Infrastruktur aktualisieren. Unternehmen sollten auch in die Schulung ihrer Mitarbeitenden investieren, damit diese die Technologie effektiver nutzen können. So wird sichergestellt, dass alle Zugang zu den Tools haben, die für die Ausführung bestimmter Aufgaben (wie die Verarbeitung von Kundendaten) benötigt werden.
```
Comment (andyundso) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:24]:
```suggestion
Die Verwaltung von Talenten und der Aufbau digitaler Fähigkeiten sind zwei Möglichkeiten, wie sich Unternehmen auf zukünftiges Wachstum vorbereiten können. Das ist unabhängig davon, ob sie die Implementierung einer komplexen Technologie wie AI planen oder lediglich ihre IT-Infrastruktur aktualisieren. Unternehmen sollten auch in die Schulung ihrer Mitarbeitenden investieren, damit diese die Technologie effektiver nutzen können. So wird sichergestellt, dass alle Zugang zu den Tools haben, die für die Ausführung bestimmter Aufgaben (wie die Verarbeitung von Kundendaten) benötigt werden.
```

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+Finden Sie die wichtigsten Kriterien für die Auswahl eines Partners: Stellen Sie sicher, dass Sie klare Anforderungen an die einzelnen Anbieter haben, bevor Sie sich für einen Anbieter entscheiden. So werden Sie später keine Überraschungen erleben, wenn Sie Zeit damit verbringen, die Produkte oder Dienstleistungen der anderen Anbieter zu bewerten.
+Erstellen Sie eine umfassende Liste potenzieller Anbieter auf der Grundlage dieser Kriterien
+Anbieter bewerten: Nehmen Sie Kontakt zu den Anbietern auf und bitten Sie sie um detaillierte Informationen über ihre Produkte oder Dienstleistungen sowie um Fallstudien von anderen Kunden, die sie erfolgreich eingesetzt haben. * Arbeiten Sie mit dem besten Anbieter für Ihre Bedürfnisse zusammen: Wenn Sie den richtigen Partner gefunden haben, ist es an der Zeit, mit ihm einen Plan für die Einführung der neuen Technologie zu erarbeiten.
+Lassen Sie sich nicht zu viel verkaufen. Lösungspartner leben davon, die Probleme als grösser darzustellen als sie sind, und vertrauen darauf dass ihre Kunden nicht genug technisches Selbstbewusstsein haben.
```
Comment (andyundso) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:33]: would good to add a list here:

```suggestion
* Verstehen Sie die Vorteile der Zusammenarbeit mit einem Technologiepartner: Ein Partner kann Ihrem Unternehmen helfen, Zugang zu den neuesten Spitzentechnologien zu erhalten, aber er verfügt auch über umfassende Erfahrung mit der Implementierung neuer Lösungen, die sich in anderen Unternehmen bewährt haben. 
* Finden Sie die wichtigsten Kriterien für die Auswahl eines Partners: Stellen Sie sicher, dass Sie klare Anforderungen an die einzelnen Anbieter haben, bevor Sie sich für einen Anbieter entscheiden. So werden Sie später keine Überraschungen erleben, wenn Sie Zeit damit verbringen, die Produkte oder Dienstleistungen der anderen Anbieter zu bewerten.
* Erstellen Sie eine umfassende Liste potenzieller Anbieter auf der Grundlage dieser Kriterien
* Anbieter bewerten: Nehmen Sie Kontakt zu den Anbietern auf und bitten Sie sie um detaillierte Informationen über ihre Produkte oder Dienstleistungen sowie um Fallstudien von anderen Kunden, die sie erfolgreich eingesetzt haben.
* Arbeiten Sie mit dem besten Anbieter für Ihre Bedürfnisse zusammen: Wenn Sie den richtigen Partner gefunden haben, ist es an der Zeit, mit ihm einen Plan für die Einführung der neuen Technologie zu erarbeiten.
* Lassen Sie sich nicht zu viel verkaufen. Lösungspartner leben davon, die Probleme als grösser darzustellen als sie sind, und vertrauen darauf dass ihre Kunden nicht genug technisches Selbstbewusstsein haben.
```

File: content/posts/die-digitale-transformation-ist-eine-reise.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+## Trends und Innovationen pragmatisch bewerten
+
+Um Ihr Unternehmen auf dem Weg der digitalen Transformation zu begleiten ist es wichtig zu verstehen, wie Trends und Innovationen helfen können. Der erste Schritt besteht darin, diese Entwicklungen aus einem pragmatischen Blickwinkel heraus zu bewerten. Sie müssen nicht nur die Technologie, sondern auch den Business Case, die Personalprozesse und die Auswirkungen auf die Kultur betrachten. Ihr Ziel sollte es sein, eine Lösung zu finden, die mit Ihrer Vision für die Zukunft übereinstimmt und gleichzeitig die aktuellen Anforderungen innerhalb eines angemessenen Zeitrahmens erfüllt.
```
Comment (andyundso) [inline content/posts/die-digitale-transformation-ist-eine-reise.md:37]:
```suggestion
Um Ihr Unternehmen auf dem Weg der digitalen Transformation zu begleiten, ist es wichtig zu verstehen, wie Trends und Innovationen helfen können. Der erste Schritt besteht darin, diese Entwicklungen aus einem pragmatischen Blickwinkel heraus zu bewerten. Sie müssen nicht nur die Technologie, sondern auch den Business Case, die Personalprozesse und die Auswirkungen auf die Kultur betrachten. Ihr Ziel sollte es sein, eine Lösung zu finden, die mit Ihrer Vision für die Zukunft übereinstimmt und gleichzeitig die aktuellen Anforderungen innerhalb eines angemessenen Zeitrahmens erfüllt.
```

**Review by andyundso — COMMENTED** [review]
I have a few suggestion, but good read overall!

**Review by Myrthi74 — APPROVED** [review]

**Pull request merged**